cwrapper/CArachneWrapperCTest.c

```diff
@@ -175,13 +129,18 @@ main(int argc, char** argv) {
     mainArgs.argc = argc;
     mainArgs.argv = argv;
 
-    retval = startCoreArbiter();
-    if (retval == -1) {
-        fprintf(stderr, "Faild to start CoreArbiter server! \n");
-        exit(1);
+    system("../CoreArbiter/bin/coreArbiterServer > /dev/null 2>&1 &");
+
+    /* Wait at most 10 sec for Arachne to initialize */
```

Comment:
Do we want to say that we're waiting specifically for Arachne to connect to the core arbiter?